config.json

```diff
@@ -234,6 +234,19 @@
           "switch-statement",
           "constructors"
         ]
+      },
+      {
+        "slug": "captains-log",
+        "name": "Captain's Log",
+        "uuid": "1ade8233-7a73-4fd9-afe2-f80d7cca14ab",
+        "concepts": [
+          "randomness"
+        ],
+        "prerequisites": [
+          "arrays",
+          "numbers",
+          "strings"
```

Comment:
Is `char` explained in `strings`?

Reply:
Technically not, although the only place where chars play a role is in the first task, but since the stub provides an array of valid planet classes technically students are not interacting with chars directly. Therefore I chose not to add it as a prerequisite. If your opinion on this differs then please let me know!

Reply:
It's probably fine